ios/Podfile

```
@@ -2,6 +2,11 @@ require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/react-native-unimodules/cocoapods.rb'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
+# Import Ruby functions from native SDK to more easily allow installing non-production SDK builds. 
+require 'open-uri'
+IO.copy_stream(URI.open('https://raw.githubusercontent.com/customerio/customerio-ios/levi/sdk-wrapper-override-script/scripts/cocoapods_override_sdk.rb'), "/tmp/override_cio_sdk.rb")
```

```suggestion
IO.copy_stream(URI.open('https://raw.githubusercontent.com/customerio/customerio-ios/main/scripts/cocoapods_override_sdk.rb'), "/tmp/override_cio_sdk.rb")
```

Once [this PR gets merged](https://github.com/customerio/customerio-ios/pull/292), this suggestion should be committed.